### PR TITLE
Cleanup Qbs file (style, formatting, copy/paste error)

### PR DESCRIPTION
**Pull request by johnbivens**
Minor changes, but for consistency's sake.

### CODE REVIEW

File: sortfilterproxymodel.qbs [hunk 1]
```diff
@@ -1,56 +1,62 @@
...
-            defines.push('QT_NO_DEBUG_OUTPUT=1');
+            defines.push('QT_NO_DEBUG_OUTPUT=1')
         }
         
-        return defines;
+        return defines
     }
 
-    cpp.includePaths: ["."];
+    cpp.includePaths: [ "." ]
```
Comment (MarcelBlanck) [inline sortfilterproxymodel.qbs:33]: I note another unwritten rule: js array with one item can be in one line?

File: sortfilterproxymodel.qbs [hunk 2]
```diff
@@ -1,56 +1,62 @@
...
     
-    Depends { name: "cpp"; }
-    Depends { name: "Qt"; submodules: product.qtModules; }
-
+    Depends { name: "cpp" }
+    Depends { name: "Qt"; submodules: product.qtModules }
```
Comment (MarcelBlanck) [inline sortfilterproxymodel.qbs:36]: And Depends object parameters can be set in one line, ok

File: sortfilterproxymodel.qbs [hunk 3]
```diff
@@ -1,56 +1,62 @@
...
 
     cpp.defines:
     {
-        var defines = [];
+        var defines = []
         
-        defines.push('QT_NO_KEYWORDS');
-        defines.push('QRMVISUALCONFIG_LIBRARY_BUILD');
+        defines.push('QT_NO_KEYWORDS')
+        defines.push('SORT_FILTER_PROXY_MODEL_LIBRARY_BUILD')
```
Comment (MarcelBlanck) [inline sortfilterproxymodel.qbs:23]: I assume it changed in code too or was not used at all

Comment (MarcelBlanck) [reply sortfilterproxymodel.qbs:23]: Ah qbs was copy paste form visual config

**Review by MarcelBlanck — APPROVED** [review]

**Comment by MarcelBlanck** [comment]
Upps I reacted on the mail but now see I was not assignee @saidinesh5 I leave the merge and review up

File: sortfilterproxymodel.qbs (hunk 1 above)
Comment (johnbivens) [reply sortfilterproxymodel.qbs:33]: We don't have any set rules for this, I was just trying to keep things as consistent as possible with the most recent Qbs files we've been writing. Most of the style "decisions" there have been made by @crozeg, but they're not really rules. Just aiming for consistency.

File: sortfilterproxymodel.qbs (hunk 2 above)
Comment (johnbivens) [reply sortfilterproxymodel.qbs:36]: (see my comment above)

File: sortfilterproxymodel.qbs (hunk 1 above)
Comment (saidinesh5) [reply sortfilterproxymodel.qbs:33]: Last week I looked at a bunch of code formatters. astyle and clang-format I think match a lot of our requirements. At least in letting us specify these spaces/indentation issues.

For QML I found none though. Maybe once we are done with C++ code formatting.. we can look into the QML one, by using/editing an existing JS formatter (there are tons of these btw.)

~1% of the commits in avalon currently are simple style fixes btw.
````
$ git log --pretty=oneline | wc -l
2400
$git log --pretty=oneline | grep -i style | wc -l
20
````

**Review by saidinesh5 — APPROVED** [review]
LGTM

**Pull request merged**